plotting/setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+sudo apt-get update
```

Comment:
(finally a code comment)
does the install need sudo?

Reply:
installing r-base or running apt-get update requires sudo on my machine at least.